protected/views/gp/index.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/* @var $this GpController */
+/* @var $dataProvider CActiveDataProvider */
+$dataProvided = $dataProvider->getData();
+$this->pageTitle = 'Practitioners';
+?>
+
+<h1 class="badge">Practitioners</h1>
+
+<div class="row data-row">
+  <div class="large-8 column">
+    <div class="box generic">
+      <table id="gp-grid" class="grid">
+        <thead>
+        <tr>
+          <th>Name</th>
+          <th>Telephone</th>
+        </tr>
+        </thead>
+        <tbody>
+        <?php foreach ($dataProvided as $gp): ?>
+          <tr id="r<?php echo $gp->id; ?>" class="clickable">
+            <td><?php echo $gp->getCorrespondenceName(); ?></td>
+            <td><?php echo $gp->contact->primary_phone; ?></td>
+          </tr>
+        <?php endforeach; ?>
+        </tbody>
+      </table>
+    </div>
+  </div>
+  <?php if (Yii::app()->user->checkAccess('TaskCreateGp')):?>
```

Comment:
Could you run auto-format on this file, and see if there are any changes? I think there should be a space between the : and the ?